modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -121,6 +121,9 @@ Couchbase Cloud Sample::
 +
 --
 If you are connecting to https://docs.couchbase.com/cloud/index.html[Couchbase Cloud], be sure to get the correct endpoint as well as user, password, and `couchbasecloudbucket`  -- and see the <<cloud-connections, Cloud section>>, below.
+During development, you may wish to use `.securityConfig(SecurityConfig.enableTls(true)` (commented out in the code sample below), 
```

Comment:
this does not make sense? enable tls is always needed both in dev and prod, I think you wanted to mention the insecure manager?